src/Pages/ContactPage.jsx

```diff
@@ -24,7 +24,8 @@ const ContactPage = () => {
         <HeroPanel title={pageTitle} />
         <Box>
           <Typography variant="h5" className={classes.subtext}>
-            Have questions? Send us a message and we’ll get
+            Have questions? Send us an email at{" "}
```

Comment:
Could we update this to `Have questions? Send us a message and we’ll get back to you within 24 hours.`? Assuming the contact form will be maintained after this merge

Reply:
Ok updated text. I think I must have misinterpreted the merge conflict when fixing that one. I thought having the link was newer. It should be fixed now.